iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -403,6 +411,33 @@ def createViewer(self, scriptUrl=DEFAULT_VIEWER_PATH, useStubViewer=False, showV
             iv.addFilingDoc(filingDocZipPath)
             self.taxonomyData["filingDocuments"] = filingDocZipName
 
+        localDocs = defaultdict(set)
+        for path, doc in dts.urlDocs.items():
+            if isHttpUrl(path):
+                continue
+            if doc.type == Type.INLINEXBRL:
+                localDocs[doc.basename].add('inline')
+            elif doc.type == Type.INLINEXBRLDOCUMENTSET:
+                localDocs[doc.basename].add('docset')
+            elif doc.type == Type.SCHEMA:
+                localDocs[doc.basename].add('schema')
+            elif doc.type == Type.LINKBASE:
+                for child in doc.xmlRootElement.iterchildren():
+                    if child.qname == XbrlConst.qnLinkCalculationLink:
+                        localDocs[doc.basename].add('calcLinkbase')
+                    if child.qname == XbrlConst.qnLinkDefinitionLink:
+                        localDocs[doc.basename].add('defLinkbase')
+                    elif child.qname == XbrlConst.qnLinkLabelLink:
+                        localDocs[doc.basename].add('labelLinkbase')
+                    elif child.qname == XbrlConst.qnLinkPresentationLink:
+                        localDocs[doc.basename].add('presLinkbase')
+                    elif child.qname == XbrlConst.qnLinkReferenceLink:
+                        localDocs[doc.basename].add('refLinkbase')
```

Comment:
I wonder if we should add a final `else .add('linkbase')` and corresponding "Other Linkbases" (possibly only enabled when non-zero)?

Reply:
Done. I suppose it's worth pointing out these are all optional. The various file summary sections only display [if there are files under them](https://github.com/austinmatherne-wk/ixbrl-viewer/blob/a314f360b795cf6780d0b09511719692a1545076/iXBRLViewerPlugin/viewer/src/js/inspector.js#L446).